src/views/token-swap/components/selected-quote/selected-quote.style.js

```diff
@@ -61,16 +61,6 @@ const useSelectedQuoteStyles = createUseStyles((theme) => ({
     color: theme.palette.grey.main,
     fontWeight: theme.fontWeights.bold,
   },
-  reward: {
```

Comment:
this is from token swap, so it should stay. It's' a different kind of reward

Reply:
Thx! Reverted!